schema/src/main/scala/org/constellation/schema/NodeState.scala

```diff
@@ -29,6 +30,8 @@ object NodeState extends Enum[NodeState] with CirceEnum[NodeState] {
 
   val invalidForJoining: Set[NodeState] = Set(Leaving, Offline)
 
+  val initiallyJoining: Set[NodeState] = Set(PendingDownload, ReadyForDownload)
```

Comment:
what does it mean that node is initially joining? is it the very first join to the first node? node cannot join again if it joined once (at least not now without clearing all the services)

Reply:
I meant it as the moment when node is joining the cluster initially, so it wasn't in the cluster a while ago. As opposed to joining other nodes in return when they are joining the cluster initially. It's a little bit of an approximation - currently we are not 100% sure that the node joined every other node when it's in DownloadInProgress/Ready and even if nodes see it's joining height, we still don't know if all the nodes have it or not and whether it's still joining or failed.